src/menu/actions.c

```diff
@@ -22,6 +22,8 @@ static void actions_clear (menu_t *menu) {
     menu->actions.options = false;
     menu->actions.settings = false;
     menu->actions.lz_context = false;
+    menu->actions.previous_tab = false;
```

Comment:
Why not just use the dpad / joystick e.g.
```
    menu->actions.go_left = false;
    menu->actions.go_right = false;
```
as already defined?

Reply:
Left, Right on the D-Pad feels more accident prone, I'd personally want something more specific for the user to press like the C buttons.
I still think that left and right on the dpad should be the fast movement.